documentation/general/contributing.md

```diff
@@ -0,0 +1,177 @@
+# Welcome
+
+Thank you for your interest in Openverse! We're so excited to bring new
+contributors into the fold in Openverse, WordPress and FOSS in general. This
+document is a set of guidelines to help you contribute to this project.
+
+## Code of Conduct
+
+You should read and agree to abide by the
+[code of conduct](https://github.com/WordPress/openverse/blob/main/CODE_OF_CONDUCT.md)
+before contributing to WordPress projects. This applies to all Openverse
+repositories because Openverse is a WordPress Foundation project.
+
+## Keep in touch
+
+Don't hesitate to ask for help; if you're stuck, we're here for you! You can
+ping us via any of our
+[communication channels](https://github.com/WordPress/openverse#keep-in-touch).
+
+## Code and code-related contributions
+
+### Get started
+
+Detailed help for contributing code can be found in the
+[developer documentation](https://docs.openverse.org/), which also includes the
+following subfolders.
+
+The following resources are preferred reading for starting your Openverse
+contribution journey in the code and code-related space.
+
+- [GitHub contribution practices](/meta/github_contribution_practices.html)
+- [Dev flow](/meta/dev_flow.html)
+- [General setup guide](/general/general_setup.html)
+- [Quickstart guide](/general/quickstart.html)
```

Comment:
Saw some more warnings in the documentation build logs. 
<img width="779" alt="Screenshot 2023-05-07 at 12 08 16 PM" src="https://user-images.githubusercontent.com/16580576/236665668-23cabac9-702b-471a-9248-4cf447542d1c.png">

Apparently the links should point to the raw Markdown files and not HTML pages.

```suggestion
- [GitHub contribution practices](/meta/github_contribution_practices.md)
- [Dev flow](/meta/dev_flow.md)
- [General setup guide](/general/general_setup.md)
- [Quickstart guide](/general/quickstart.md)
```